### ISSUE TITLE
problem with v3.4.1 summary HTML page not showing data

### ISSUE THREAD

**Issue by bblarsen-sci** [dms-vep/dms-vep-pipeline-3#58]
@jbloom I am trying to run the new v3.4.1 of pipeline to show summaries of selections for mAB, entry, and receptor affinity. I modified my `summaries_config.yml` file based on the one in Bernadeta's XBB repo. The pipeline runs fine with no errors, but when I click to view the [summary page](https://dms-vep.github.io/Nipah_Malaysia_RBP_DMS/htmls/summary_overlaid.html), it only shows the sliders and not the heatmap data. When I look at the raw .html file the data all appear to be in there, so I'm not sure what the issue is.

**Comment by jbloom** [comment]
@bblarsen-sci, this is fixed in version 3.4.3.

Basically, the issue is that having antibody names with a `.` in them causes some sort of strange internal `altair` error. I can't easily change it to allow names like that, but I have modified pipeline so it now throws and interpretable error rather than silently making an empty plot.

Just change the relevant lines in `summaries_config.yml` to read something like this when defining the antibody names for plotting:
```
      antibody_list:  # antibodies to include: keys name used for `avg_escape`, values name for plot
        nAH1.3: nAH1_3
        m102.4: m102_4
        HENV32: HENV32
        HENV117: HENV117
        HENV103: HENV103
        HENV26: HENV26
        minibinder: minibinder
```

**Comment by bblarsen-sci** [comment]
Great, thank you.
